src/extension.ts

```diff
@@ -225,6 +232,15 @@ export async function activate(context: vscode.ExtensionContext) {
   return {
     loadLanguage,
 
+    registerLanguage(languageId, wasm) {
```

Comment:
I think I'd call this `wasmPath` and put a quick note in a docstring to indicate that the path has to be absolute